src/main/java/org/broadinstitute/consent/http/mail/SendGridAPI.java

```diff
@@ -94,7 +93,7 @@ private boolean findUserEmailPreference(Mail message) {
       if (!tos.isEmpty()) {
         User user = userDAO.findUserByEmail(tos.get(0).getEmail());
         if (Objects.isNull(user)) {
-          logger.error("Unknown user: " + tos.get(0).getEmail());
+          logWarn("Unknown user: %s".formatted(tos.get(0).getEmail()));
```

Comment:
should this be logException or do we want to change it to warn?

Reply:
Ah yes, this was an intentional change - I should have commented on that. `logException` triggers a sentry alert which is overkill for this kind of problem.